app/assets/javascripts/components/features/ui/components/onboarding_modal.jsx

```diff
@@ -90,26 +93,26 @@ const PageFour = ({ domain, intl }) => (
       <div className='row'>
         <div>
           <div className='figure non-interactive'><ColumnHeader icon='home' type={intl.formatMessage(messages.home_title)} /></div>
-          <p><FormattedMessage id='onboarding.page_four.home' defaultMessage='Home timeline shows posts from people you follow'/></p>
+          <p><FormattedMessage id='onboarding.page_four.home' defaultMessage='The Home timeline shows posts from people you follow'/></p>
         </div>
 
         <div>
           <div className='figure non-interactive'><ColumnHeader icon='bell' type={intl.formatMessage(messages.notifications_title)} /></div>
-          <p><FormattedMessage id='onboarding.page_four.notifications' defaultMessage='Notifications show when someone interacts with you' /></p>
+          <p><FormattedMessage id='onboarding.page_four.notifications' defaultMessage='The Notifications column shows when someone interacts with you' /></p>
```

Comment:
While we're changing these, what do you think about adding periods to the end of this sentence, and to the end of the explanation for Home feeds as well? It would make it consistent with the rest of the copy text.